### PR TITLE
Bash implementation

### PR DESCRIPTION
**Pull request by ejholmes**
This is a small bash implementation of the readme.

### CODE REVIEW

File: docker-build [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+REPO="$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME"
+
+# Usage prints the help for this command.
+usage() {
+  echo "Usage:"
+  echo "    docker-build command"
+  echo ""
+  echo "Commands:"
+  echo "    build  Build a docker image from this repo and tag it with the git sha and branch"
+  echo "    push   Push the built image to the docker registry"
```
Comment (phobologic) [inline docker-build]: Maybe put a comment in here about all the variables that are necessary for this to run?

**Comment by phobologic** [comment]
:+1:

**Comment by ejholmes** [comment]
Added branch tagging to this as well.

File: docker-build (hunk 1 above)
Comment (ejholmes) [inline docker-build]: Good call, I'll have it error out of any of the DOCKER_ variables aren't present.

**Pull request merged**